hyperon_das/utils.py

```diff
@@ -175,50 +178,98 @@ def connect_to_server(host: str, port: int) -> Tuple[int, str]:
 
 
 def check_server_connection(url: str) -> Tuple[int, str]:
-    logger().debug(f'connecting to remote Das {url}')
-    error_msg = None
+    logger().debug(f"Connecting to remote DAS {url}")
+
     try:
         das_version = get_package_version('hyperon_das')
         atom_db_version = get_package_version('hyperon_das_atomdb')
 
         with sessions.Session() as session:
             payload = {
-                'action': 'handshake',
-                'input': {
-                    'das_version': das_version,
-                    'atomdb_version': atom_db_version,
-                },
+                "action": "handshake",
+                "input": {},
             }
-            response = session.request(
-                method='POST',
+            response = session.post(
                 url=url,
                 data=serialize(payload),
-                headers={'Content-Type': 'application/octet-stream'},
+                headers={"Content-Type": "application/octet-stream"},
                 timeout=10,
             )
-        if response.status_code == HTTPStatus.CONFLICT:
-            try:
-                response = deserialize(response.content)
-                remote_das_version = response.get('das').get('version')
-                remote_atomdb_version = response.get('atom_db').get('version')
-            except JSONDecodeError as e:
-                raise Exception(str(e))
-            logger().error(
-                f"Package version conflict error when connecting to remote DAS. Local DAS: 'das: {das_version} - atom_db: {atom_db_version}' -- Remote DAS: 'das: {remote_das_version} - atom_db: {remote_atomdb_version}'"
+
+        if response.status_code != HTTPStatus.OK:
+            response.raise_for_status()
```

Comment:
You don't need to check for status here, the raise_for_status() will take care of it.
```suggestion
        response.raise_for_status()
```

Reply:
you're right, thank you